verify.js

```diff
@@ -179,6 +179,12 @@ module.exports = function (jwtString, secretOrPublicKey, options, callback) {
       }
     }
 
+    if (options.nonce) {
```

Comment:
Could you return error if `options.nonce` is not a string?

Reply:
With the above validation is added, then we should test that the following values result in the error being returned:

```
[
  true,
  false,
  null,
  -1,
  0,
  1,
  -1.1,
  1.1,
  -Infinity,
  Infinity,
  NaN,
  '',
  ' ',
  [],
  ['foo'],
  {},
  {foo: 'bar'},
]
```